File: scala/src/main/scala/HelloWorld.scala
import java.util.Scanner

object HelloWorld {
  def main(args: Array[String]): Unit = {
    val sc = new Scanner(System.in)
    val cases = sc.nextInt

    for (_ <- 0 until cases) {
      val name = sc.next()
      println(s"Hello, $name!")
    }
  }
}

File: java/src/main/java/HelloWorld.java
import java.util.Scanner;

class HelloWorld {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int cases = sc.nextInt();

        for (int i = 0; i < cases; i++) {
            String name = sc.next();
            System.out.println("Hello, " + name + "!");
        }
    }
}
